### DISCUSSION TITLE
Recommended Server Specs

### DISCUSSION THREAD

**Discussion by bruno-robert** [EnigmaticaModpacks/CreateTogether#7] [Q&A] [answered]
What are the recommended server specifications for this mod pack?
RAM, CPU cores, etc.

**Comment by NielsPilgaard** [comment, marked as answer]
Depends on the amount of concurrent players you'll have :)

**Reply by bruno-robert** [reply]
I'm thinking about 2-5 players 🤔

**Reply by NielsPilgaard** [reply]
You don't need much, I'd say 3GB should be just fine :)
